### ISSUE TITLE
Broken build after 

### ISSUE THREAD

**Issue by rgov** [stp/stp#301]
Commit 80742b4858e695e741a30f6d4111ee63e6917e9c broke the build: https://travis-ci.org/stp/stp/builds/428358275

**Comment by conp-solutions** [comment]
I applied the proposed the above fix to RIss' cmake. In case this change passes travis, I'm happy to push the change, so that the STP build works again:
https://github.com/conp-solutions/riss/pull/8

Eventually, during the STP build, solvers should be picked based on commit ID (similarly to git submodules), so that the case "something changed in the other solver" can be ruled out.

**Comment by msoos** [comment]
@conp-solutions  I agree about the commit-ID stuff, actually :) Can you please add that to the travis-ci script that git clones Riss? It's file "travis-cmake.sh" and the offending code is:

```
git clone --depth 1 https://github.com/conp-solutions/riss.git
```

You know best what to change it to. I'd be really glad if you did it. Thanks!

**Comment by conp-solutions** [comment]
I added the memaccess check to Riss' build system. Can you retry the build now please?

Before I look into the above commit-ID stuff, there is other things in Riss' CI that fail and that I have to root cause before declaring a "good version" to be always used by STP.

**Comment by rgov** [comment]
Build is green now.

![](https://travis-ci.org/stp/stp.svg?branch=master)
